masterlist.yaml

```diff
@@ -6283,22 +6283,20 @@ plugins:
       - crc: 0xC32BA914
         util: 'SSEEdit v4.0.3'
 
-  - name: 'TouringCarriages.esp'
+  - name: 'TouringCarriages(-ICAIO)?.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/15948/' ]
+  - name: 'TouringCarriages.esp'
     after:
       - 'Landscape Fixes For Grass Mods.esp'
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
       - 'ViscousGrass.esp'
-    tag: [ Keywords ]
+    tag: [ C.Location ]
     clean:
-      - crc: 0x15B143C8
+      - crc: 0xC48E340B
         util: 'SSEEdit v4.0.3'
   - name: 'TouringCarriages-ICAIO.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/15948/' ]
     after: [ 'Immersive Citizens - TC patch.esp' ]
-    req:
-      - 'Immersive Citizens - AI Overhaul.esp'
-      - 'TouringCarriages.esp'
+    req: [ 'Immersive Citizens - AI Overhaul.esp' ]
```

Comment:
Yeah that's fine, the first version of the patch didn't have either as a master file, but it's gone now.

Reply:
Figured as much.